### PR TITLE
Add support for insecure apiserver

### PR DESCRIPTION
**Pull request by utkarshmani1997**
1. Why is this change necessary ?
- It was failing if using flag insecure in kube-apiserver

2. How does this change address the issue ?
- Adds flag to pass master-ip or kube-config to be passed as command and
  arg in yaml file

3. How to verify this change ?
- Run `kubectl create -f k8s-deployment.yaml`

4. What side effects does this change have ?
- none

5. Other details

### CODE REVIEW

File: k8s-client/_vendor-20180202164612/github.com/PuerkitoBio/purell/purell.go [hunk 1]
```diff
@@ -0,0 +1,379 @@
+/*
+Package purell offers URL normalization as described on the wikipedia page:
+http://en.wikipedia.org/wiki/URL_normalization
+*/
```
Comment (prateekpandey14) [inline k8s-client/_vendor-20180202164612/github.com/PuerkitoBio/purell/purell.go]: remove this

**Comment by kmova** [comment]
Thanks @utkarsh -- can you and @amitd -- clean up the user display messages

File: k8s-client/listpods.go [hunk 2]
```diff
@@ -18,18 +18,36 @@ limitations under the License.
...
+)
+
 func main() {
+	flag.Parse()
 	// creates the in-cluster config
-	config, err := rest.InClusterConfig()
+	var config *rest.Config
+	var err error
+	if *master != "" || *kubeconfig != "" {
+		fmt.Printf("Either master or kubeconfig specified. building kube config from that..\n")
```
Comment (AmitKumarDas) [inline k8s-client/listpods.go]: Move this statement after actual execution.
The message can be "Client config was built using flags: Address: '%s' Kubeconfig: '%s' \n"

**Comment by ksatchit** [comment]
K8s-client is not included in the latest repo structure on litmus. Closing this PR.

**Pull request closed without merging**